tests/unit/data_formats/datumaro/test_datumaro_format.py

```diff
@@ -175,6 +190,10 @@ def test_can_save_and_load(
     def test_source_target_pair(
         self, fxt_dataset_pair, compare, require_media, dimension, test_dir, helper_tc, request
     ):
+        if fxt_dataset_pair == "fxt_point_cloud_dataset_pair" and sys.platform == "win32":
```

Comment:
This bug is related to point cloud and it needs to be revisited to resolve failures on windows.

Reply:
This is fixed by [4caf468](https://github.com/openvinotoolkit/datumaro/pull/1547/commits/4caf468495e0c34875a88dbb35de68ae2bf3fe2e)